src/store/Store.js

```diff
@@ -47,36 +47,40 @@ class Store {
     this.readyPromise = new Promise(resolve => this.readyResolve = resolve);
 
     this.browserAPI = getBrowserAPI();
-    this.extensionId = extensionId; // keep the extensionId as an instance variable
-    this.port = this.browserAPI.runtime.connect(this.extensionId, {name: portName});
-    this.safetyHandler = this.safetyHandler.bind(this);
-    if (this.browserAPI.runtime.onMessage) {
-      this.safetyMessage = this.browserAPI.runtime.onMessage.addListener(this.safetyHandler);
-    }
-    this.serializedPortListener = withDeserializer(deserializer)((...args) => this.port.onMessage.addListener(...args));
-    this.serializedMessageSender = withSerializer(serializer)((...args) => this.browserAPI.runtime.sendMessage(...args), 1);
+    this.initializeStore = this.initializeStore.bind(this);
+
+    // We request the latest available state data to initialise our store
+    this.browserAPI.runtime.sendMessage(
+      { type: FETCH_STATE_TYPE, portName }, undefined, this.initializeStore
```

Comment:
I know I was responsible for not doing it 😅, but now that it will become part of the main package, I wonder if we are drooping port, and if it would make sense to rename `portName` to a more accurate representation.

Reply:
Good point, it would make sense to rename. The first name that comes to mind is `messageType`, any other ideas?

Reply:
Ended up renaming to `channelName`